config/vufind/BrowZine.ini

```
@@ -67,6 +72,20 @@ browzineWebLink = "View Complete Issue|browzine-issue|https://assets.thirdiron.c
 fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg"
 retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
 
+; The bestIntegrationLink service is not yet documented on that wiki page, but it provides
```

Is this a typo, or a distinction that I'm missing?

```suggestion
; The bestIntegratorLink service is not yet documented on that wiki page, but it provides
```